### PR TITLE
fix github_tag_version.py annotated tagging

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

File: codekit/cli/github_mv_repos_to_team.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def main():
         print len(move_me), 'repos to me moved'
 
     # FIXME unused
-    # teams = [g for g in org.iter_teams()]
+    # teams = [g for g in org.teams()]
```
Comment (jonathansick) [inline codekit/cli/github_mv_repos_to_team.py]: might as well just delete this

File: codekit/cli/github_tag_version.py [hunk 2]
```diff
@@ -46,6 +46,9 @@ def parse_args():
         github-tag-version --org lsst --team 'Data Management' \
             --team 'External' --candidate v11_0_rc2 11.0.rc2 b1679
 
+        Note that the access token must have access to these scopes:
```
Comment (jonathansick) [inline codekit/cli/github_tag_version.py]: Nice.

Does codekit's `github-auth -u {{username}}` command also get this properly. You might drop a note here that you can get the token with that command, but I can see why you'd want to also manually specify the token's properties.

File: codekit/cli/github_tag_version.py [hunk 3]
```diff
@@ -212,7 +215,12 @@ def main():
             print '!!! SKIPPING', upstream, (60-len(upstream)) * '-'
             continue
 
-        for team in repo.iter_teams():
+        if not sum(1 for _ in repo.teams()):
```
Comment (jonathansick) [inline codekit/cli/github_tag_version.py]: this is a bit special :) Is repo.teams() an iterator and that's why you can't test length?

Comment (jhoblitt) [inline codekit/cli/github_tag_version.py]: Aye -- the method returns an iterator.  I wrestled with trying to make the expression more intelligible but my python foo failed me.

File: codekit/cli/github_tag_version.py (hunk 2 above)
Comment (jhoblitt) [inline codekit/cli/github_tag_version.py]: Yes, `github-auth` will requested the appropriate scopes.  I'll add a note.

**Pull request merged**